### PR TITLE
Google Platforms - Case Study

### PR DESCRIPTION
**Pull request by joakimlees**
Would like some feedback on my case-study.

Are we suppose to format the text file in a certain way? like different from what I have done, or is this fine?

### CODE REVIEW

File: google-platforms-case-study.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+keywords: development platforms, google analytics, analytics, GA4, GTM, tracking, google tag manager
+tags: Development Platforms, google platforms, analytics
+sidebar: development-platforms
+permalink: development-platforms/google-platforms.html
+folder: development-platforms
+---
+
+## Introduction
+
+This study mainly covers Google analytics, but will also touch on Google tag manager, as it is a very related platform.
```
Comment (Fermain) [inline google-platforms-case-study.md:12]:
```suggestion
This study mainly covers Google analytics, but will also touch on Google tag manager, as it is a closely related platform.
```

File: google-platforms-case-study.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+Google Analytics is an analysis tool, which is used to gain insights and provide reports on user behavior on websites and/or web applications. Since Universal google analytics(UA) expires in July 2023 (Weber, 2023), this study will cover the already exciting successor google analytics 4(GA4).
```
Comment (Fermain) [inline google-platforms-case-study.md:14]:
```suggestion
Google Analytics is an analysis tool used to gain insights and provide reports on web application user behaviour. Since Universal google analytics (UA) expires in July 2023 (Weber, 2023), this study will cover the already exciting successor, google analytics 4 (GA4).
```

Comment (Fermain) [reply google-platforms-case-study.md:14]: Please make sure (brackets) have a space on either side.

File: google-platforms-case-study.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+Google tag manager is, as the name suggests, a tag management tool. The purpose is to make it easy to manage tracking codes for various platforms, not only google analytics but also platforms such as google ads, Meta platforms/facebook pixel etc. (Fedorovicius, 2022)
```
Comment (Fermain) [inline google-platforms-case-study.md:16]: What is a tag?

File: google-platforms-case-study.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+- 2005: 1st release. Urchin from Google(urchin.js). (Franco & Valdés, 2021)
+- November 2005: Google Introduced the first version of Google Analytics. (digitalstateconsulting.com, 2014).
+
+- 2007: 2nd release. Classic Google Analytics: Google Analytics Synchronous Code(ga.js). (Franco & Valdés, 2021)
+- 2007: The Analytics’ reporting interface was revamped for “greater customization and collaboration. "moving analytics from the back room of techies into the boardroom" - (digitalstateconsulting.com, 2014).
+
+- 2008: Continued to evolve with a series of beta updates. Such as: Advanced Segmentation, Custom Reports, Multi-dimensional Motion Charts, a new Management Dashboard. (digitalstateconsulting.com, 2014).
+
+-2009: 2nd release update. Google Analytics Asynchronous Code (ga.js). (Franco & Valdés, 2021)
```
Comment (Fermain) [inline google-platforms-case-study.md:34]:
```suggestion
- 2009: 2nd release update. Google Analytics Asynchronous Code (ga.js). (Franco & Valdés, 2021)
```

File: google-platforms-case-study.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+Google tag manager is, as the name suggests, a tag management tool. The purpose is to make it easy to manage tracking codes for various platforms, not only google analytics but also platforms such as google ads, Meta platforms/facebook pixel etc. (Fedorovicius, 2022)
+
+## Brief History
+
+"Google Analytics is by far the biggest web analytics platform in the world" - Djuraskovic, O (2022)
+
+- 1998: A statistics analysis program, Urchin was developed.
+- March 2005: Google acquired Urchin.
+  (digitalstateconsulting.com, 2014)
```
Comment (Fermain) [inline google-platforms-case-study.md:24]: Move onto the line above

File: google-platforms-case-study.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
```
Comment (Fermain) [inline google-platforms-case-study.md:25]: Remove whitespaces between bullets

File: google-platforms-case-study.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+- 2007: 2nd release. Classic Google Analytics: Google Analytics Synchronous Code(ga.js). (Franco & Valdés, 2021)
+- 2007: The Analytics’ reporting interface was revamped for “greater customization and collaboration. "moving analytics from the back room of techies into the boardroom" - (digitalstateconsulting.com, 2014).
+
+- 2008: Continued to evolve with a series of beta updates. Such as: Advanced Segmentation, Custom Reports, Multi-dimensional Motion Charts, a new Management Dashboard. (digitalstateconsulting.com, 2014).
+
+-2009: 2nd release update. Google Analytics Asynchronous Code (ga.js). (Franco & Valdés, 2021)
+
+- 2009: A new AdSense integration feature which helped users to measure website performance and ROI. In addition to a universal rollout of the Data Export API, which supported the integration of web analytics across multiple platforms.
+  October 2009: Google released version 4 og google analytics, with features such as: New algorithm, profile conversions, create goals for time spent on site and page views per visit.
```
Comment (Fermain) [inline google-platforms-case-study.md:37]:
```suggestion
- October 2009: Google released version 4 og google analytics, with features such as: New algorithm, profile conversions, create goals for time spent on site and page views per visit.
```

File: google-platforms-case-study.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+- In late 2011: Google launched its real time report feature
+- October 2011: GA introduced flow visualizations.
+- October 2012: Google’s official announcement of Universal analytics(UA)
+  (digitalstateconsulting.com, 2014)
+
+- 2012: 3rd release. Google Analytics Tag (analytics.js). (Franco & Valdés, 2021)
+
+- 2013: More than 70 product updates for UA. Such as improved user interface, new APIs, more real-time reports for users better to interpret visitor behavior. (digitalstateconsulting.com, 2014).
+
+-2017: 3rd release update. Global Site Tag (gtag.js). (Franco & Valdés, 2021)
```
Comment (Fermain) [inline google-platforms-case-study.md:52]:
```suggestion
- 2017: 3rd release update. Global Site Tag (gtag.js). (Franco & Valdés, 2021)
```

File: google-platforms-case-study.md [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+In GA4 you will find four different types of engagement reports
+
+- Events, could be anything happening or user action on the page. Like: page view, button click, form submit, scrolled.
+- Conversions, is an event, but usually an event which is view as the final step in a number of events. Like: making a purchase or complete registration.
+- Pages and screens, lets you view on which page the engagement is happening. How many views a certain page has, total events triggered on a certain page or how many users visited the certain page.
+- Landing pages, gives you an overview of the engagement on the different landing pages (the first page of the website people viewed when entering the website)
+
+#### Demographics reports
+
+Lets you see from what city or country your traffic/users are coming from.
```
Comment (Fermain) [inline google-platforms-case-study.md:104]: Why might this be useful for marketing?

File: google-platforms-case-study.md [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+
+#### Exploration reports
+
+Makes you customize your own reports, which could provide greater insights to different user journeys.
+
+In GA4 you could setup a lott of complex reports, which might fit your needs better insights then whats provided by the standard reports.
```
Comment (Fermain) [inline google-platforms-case-study.md:110]:
```suggestion
In GA4 you could setup a lot of complex reports, which might fit your needs better insights than what is provided by the standard reports.
```

File: google-platforms-case-study.md [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+Lets say you wanted to view a report with the following conditions: Users who made a purchase, but only users who clicked the primary button on the home page and which did not arrive from paid ads. You could add all these filters each time on one of the standard reports, but if this is a report you would like to view more frequently, to make an exploration report for it would be much more efficient.
```
Comment (Fermain) [inline google-platforms-case-study.md:111]:
```suggestion
Let's say you wanted to view a report with the following conditions: Users who made a purchase but only users who clicked the primary button on the home page and did not arrive from paid ads. You could add all these filters each time on one of the standard reports, but if this is a report you would like to view more frequently, to make an exploration report for it would be much more efficient.
```

File: google-platforms-case-study.md [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+- Audiences, GA4 let you create different audiences, which you can customize yourself. i.e. Only users who made a purchase could be a audience.
+- Traffic filters, Filter traffic from certain ip-addresses. i.e. Internal traffic.
+- Unwanted referrals filter, Filter traffic from websites you don't want to show up in your reports.
+- Sessions, let you define what should be counted as a session(a user interacting with your website).
+- Engaged session, let you define how long time
+-
+
+### Google Tag Manager
+
+As mentioned in the introduction, I will very briefly cover the platform google tag manager(GTM) and how it works, and how it works in relation to google analytics.
```
Comment (Fermain) [inline google-platforms-case-study.md:129]: There is no **I** in technical literature (I mean, there are two, but...).

We use `the author` to refer to ourselves instead of I. In this case I think you can shorten this to avoid it completely.

Comment (Fermain) [inline google-platforms-case-study.md:129]:
```suggestion
As mentioned in the introduction, we will briefly cover the platform google tag manager (GTM) and how it works, especially in relation to google analytics.
```

File: google-platforms-case-study.md [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+
+## Strengths
+
+So what's so great about GA4, besides obviously being a great tool for digital marketers who's looking to increase their sales.
+
+How could we use it to speed up and improve our development?
+
+GA4 is great for proving insights in the form of numbers and user journeys.
+
+It provide insights on how many times somethings happening on a website, on what page it's happening on, how long people are staying, what page they previously visited, on what page users land on and what page they left, and so on.
```
Comment (Fermain) [inline google-platforms-case-study.md:145]:
```suggestion
It provides insights on how often something is happening on a website, on which page, how long users are staying, which page they previously visited, on which page users land and what page they left, and so on.
```

File: google-platforms-case-study.md [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+
+We could use this data to develope the website and further improve the user experience.
```
Comment (Fermain) [inline google-platforms-case-study.md:147]:
```suggestion
We could use this data to develop the website and further improve the user experience.
```

File: google-platforms-case-study.md [hunk 15]
```diff
@@ -0,0 +1,185 @@
...
+
+Say a lott of users left the website on a certain page, or say nearly no one clicked the primary button on the home page. Well that gives us an assumption on that something about that page or that button might not be very user friendly.
```
Comment (Fermain) [inline google-platforms-case-study.md:149]:
```suggestion
For example, if users left the website on a certain page, or say nearly no one clicked the primary button on the home page. That may lead to the conclusion that something about that page or that button might not be very user-friendly.
```

**Review by Fermain — CHANGES_REQUESTED** [review]
This is an interesting case study on an important tool, and one that is not focussed on enough in our course. I think this will be of great value to other students in future looking to demystify analytics.

The majority of my comments are either formatting, spelling or phrasing rather than content. I have left some prompts that you may find useful.

File: google-platforms-case-study.md (hunk 3 above)
Comment (joakimlees) [reply google-platforms-case-study.md]: slight adjustement on the section and added a section with a more detailed description on a tag.

```
Google tag manager (GTM) is, as the name suggests, a tag management tool, better known as tag management system (TMS). The purpose is to make it easy to manage tracking codes for various platforms, not only google analytics but also platforms such as google ads, Meta platforms/facebook pixel etc. (Fedorovicius, 2022)

The purpose of a TMS like GTM is to allow for a quick and easy way to update measurement codes and related code fragments, which collectively is known as tags, on a website or web application. This is done by applying a small segment of Tag Manager code to the project. With that code applied the developer can then safely and easily deploy analytics and measurement tag configurations from a web-based user interface. (Google Help, 2023)
```

**Comment by SaraRFerrer** [comment]
I previously took a bachelors degree in marketing and had to get certified in Google Analytics so it was really interesting to read your case study covering the topic! I think it was very well written and informative. Good job! :)

**Review by Jenfer93 — COMMENTED** [review]
Great work, Joakim! This case provided me with understanding of what Google Analytics are, what it is for, and its pros and cons. I liked the writing and the flow in it, good job :)

**Review by Fermain — APPROVED** [review]
This is a really well researched and written case study that will be useful to many. Thanks for your quality contributions!
